nautobot/extras/plugins/views.py

```diff
@@ -132,8 +143,8 @@ def get(self, request, app=None, plugin=None):
             request,
             "extras/plugin_detail.html",
             {
+                "app_data": extract_app_data(app_config, load_marketplace_data()),
                 "object": app_config,
```

Comment:
Would it make sense to include min_version and max_version in "app_data" as well? So we only need to pass in one context variable and we can leave "object" out?

Reply:
`object` is still needed because it contains all of the app "features" for the right panel of the detail view. Since that info is *only* needed for the detail view, I don't think it makes sense to populate all of it into `app_data`.